### PR TITLE
kubernetes volume concepts

### PR DESCRIPTION
**Pull request by ramkrsna**
adding the kubernetes files under /docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +20,10 @@ deployment scheme.
 technology!](https://drive.google.com/file/d/0B667S2caJiy7QVpzVVFNQVdyaVE/view?usp=sharing) <<<**
 
 
+
+
```
Comment (MohamedAshiqrh) [inline README.md]: Do we need space? I do not see any difference though. 

View : https://github.com/ramkrsna/gluster-kubernetes/tree/addDocs

File: docs/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Kubernetes Concepts for Gluster Storage
+
+[Persistent Volumes] (http://kubernetes.io/docs/user-guide/persistent-volumes/)
```
Comment (MohamedAshiqrh) [inline docs/kubernetes.md]: When you view this it looks to be in same line. can we be able to make it in new line? would be more readable.

View : https://github.com/ramkrsna/gluster-kubernetes/blob/addDocs/docs/kubernetes.md

File: README.md (hunk 1 above)
Comment (ramkrsna) [inline README.md]: No. I guess it was just the editor settings which caused it to be there

File: docs/kubernetes.md (hunk 2 above)
Comment (ramkrsna) [inline docs/kubernetes.md]: Thanks for spotting it . I have made the necessary changes.

Comment (MohamedAshiqrh) [inline docs/kubernetes.md]: looks good now :)

File: docs/kubernetes.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # Kubernetes Concepts for Gluster Storage
 
-[Persistent Volumes] (http://kubernetes.io/docs/user-guide/persistent-volumes/)
-[Persistent Volume Implementation] (https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/volume-provisioning.md)
-[Dynamic Provisioning and Storage Classes in Kubernetes] (http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html)
+*[Persistent Volumes] (http://kubernetes.io/docs/user-guide/persistent-volumes/)
```
Comment (obnoxxx) [inline docs/kubernetes.md]: The formatting is wrong: I think you need space between bullet point and text. See the result here:
https://github.com/ramkrsna/gluster-kubernetes/blob/addDocs/docs/kubernetes.md

File: README.md (hunk 1 above)
Comment (obnoxxx) [inline README.md]: please remove the changes to the README.md file. your patch should only touch the new kubernetes.md file, i guess.

**Comment by wattsteve** [comment]
I think Jose is about to submit a PR that is greatly going to simplify the deployment process and really reduces the amount of steps in the instructions

**Comment by obnoxxx** [comment]
@ramkrsna ping?

**Comment by ramkrsna** [comment]
The PR from Jose fixes most of this stuff. Additional Kubernetes Steps would add complexity

**Comment by obnoxxx** [comment]
@ramkrsna are you withdrawing this PR then? if so, please close!

**Comment by ramkrsna** [comment]
the deploy directory has the details for kubernetes . Closing the PR

**Pull request closed without merging**